app/views/compute_resources_vms/form/kubevirt/_volume.html.erb

```diff
@@ -1,19 +1,21 @@
-<% storage_classes = compute_resource.storage_classes %>
+<% if f.object.type.nil? || f.object.type == 'persistentVolumeClaim' %>
+  <% storage_classes = compute_resource.storage_classes %>
 
-<%= select_f f, :storage_class,
-                compute_resource.storage_classes_for_select,
-                :id,
-                :description,
-                { },
-                :label => _('Storage Class'),
-                :label_size => "col-md-2",
-                :disabled => !new_vm,
-                :class => "col-md-2" %>
+  <%= select_f f, :storage_class,
+                  compute_resource.storage_classes_for_select,
+                  :id,
+                  :description,
+                  { },
```

Comment:
what's the empty hash represents?

Reply:
that's the `select_options` from select_f definition